### ISSUE TITLE
Frontends cannot be started

### ISSUE THREAD

**Issue by peterboldizs** [PacktPublishing/Hands-on-Microservices-with-Python#16]
I have started the microservices using the docker-compose.deploy.yml file.
The frontendgit_order_db_1 container cannot be started because of this error:

```
Traceback (most recent call last):
  File "/app/app.py", line 1, in <module>
    from flask import Flask
  File "/usr/local/lib/python3.9/site-packages/flask/__init__.py", line 19, in <module>
    from jinja2 import Markup, escape
ImportError: cannot import name 'Markup' from 'jinja2' (/usr/local/lib/python3.9/site-packages/jinja2/__init__.py)
```

I have updated the base image as recommended in issue #15 to python:3.9-alpine in all Dockerfiles.
I have also checked that the upgrades recommended in issue #4 are already in the master branch.
Any idea how to fix this?

**Comment by peterboldizs** [comment]
I have forked all repos and tried to fix the issues. At least all services in the Docker Compose file are now able to start.

https://github.com/peterboldizs/Hands-on-Microservices-with-Python/tree/all-new-deps
https://github.com/peterboldizs/Hands-on-Microservices-with-Python-Product-Service/tree/all-new-deps
https://github.com/peterboldizs/Hands-on-Microservices-with-Python-Order-Service/tree/all-new-deps
https://github.com/peterboldizs/Hands-on-Microservices-with-Python-User-Service/tree/all-new-deps
You might want to consider these for pull requests.

I have tested all 3 APIs using Postman and they are working fine. However, the frontend still has runtime errors but I am investigating those further.
